app/models/label/LabelTable.scala

```diff
@@ -728,9 +729,21 @@ object LabelTable {
     // Init random function.
     val rand = SimpleFunction.nullary[Double]("random")
 
+    // Prioritize labels to show by filtering out those that have been
+    // validated as "disagree" 3 or more times and have twice as many
+    // disagrees as agrees
+    val prioritizedLabels = Q.queryNA[(Int)](
```

Comment:
Since the query is actually completely identical and is copied three times, could you separate it out into it's own function?

Reply:
yup